panel/src/components/Sections/ModelsSection.vue

```diff
@@ -233,6 +323,33 @@ export default {
 			this.searching = !this.searching;
 			this.searchterm = null;
 		},
+		onSelect(item) {
+			if (this.selected.includes(item)) {
+				this.selected = this.selected.filter(
+					(selected) => selected.id !== item.id
+				);
+			} else {
+				this.selected.push(item);
+			}
+		},
+		onSelectToggle() {
+			this.isSelecting ? this.stopSelecting() : this.startSelecting();
```

Comment:
Maybe we could spare ourselves also those two methods and just do

```js
this.selected = [];
this.isSelecting = !this.isSelecting;
```

Reply:
Since we already have the stopSelecting method, I find the code above more readable.

Reply:
This really depends on your comment and the decision for the created method. When we still need the stopSelecting method there, we should also keep using it in the toggle method.